README.md

```diff
@@ -2,64 +2,54 @@
 
 > ESLint [shareable config](http://eslint.org/docs/developer-guide/shareable-configs.html) developed and maintained by [nimble](https://nimblehq.co/)
 
-## Installation
+## Usage
 
-```
-$ npm install --save-dev @nimblehq/eslint-config-nimble
-```
+The configurations are separated into dedicated packages:
+- [eslint-config-nimble](/packages/eslint-config-nimble): ESLint base rules
 
-## Usage
+__Usage information is in the packages' documentation.__
 
-### Standalone
+## How to contribute
 
-Add `@nimblehq/eslint-config-nimble` to the extends section of your `.eslintrc` configuration file.
+- This project uses [Lerna](https://lerna.js.org/) to manage packages. Install Lerna for accessing to the lerna CLI.
 
-```js
-{
-  "extends": [
-      "@nimblehq/eslint-config-nimble"
-  ],
-  "rules": {
-    // Additional, per-project rules...
-  }
-}
-```
+- To contribute to the existing packages, simply navigate to the `/packages` folder and create a pull request to change them.
 
-### With a framework
+### Add a new package
+
+- Use [`lerna create`](https://github.com/lerna/lerna/tree/main/commands/create#readme) command to add a new package.
+
+- The package name should have the following format: `@nimblehq/{package-name}`. This project is shareable configurations for [ESLint](https://eslint.org/), the `{package-name}` should be `eslint-config-nimble-*`:
+
+```bash
+  npx lerna create @nimblehq/eslint-config-nimble-react
```

Comment:
```suggestion
  lerna create @nimblehq/eslint-config-nimble-react
```

If we recommend to install `lerna`, no need to use npx. Or earlier in this README, we can remov the "Install Lerna for accessing to the lerna CLI" ;-)

Reply:
I removed the npx instead 😂